src/Infrastructure/Processor/Process/MediaTranslationShopware6ExportProcess.php

```
@@ -0,0 +1,150 @@
+                    $altTranslations[$shopwareLanguage->getIso()] ?? null,
+                    $titleTranslations[$shopwareLanguage->getIso()] ?? null,
+                    $shopwareLanguage->getId()
+                );
+            }
+        }
+        $shopware6Media->updateTranslated($ergoTranslations);
+
+        if ($shopware6Media->isModified()) {
+            $this->logger->info(
+                'Updating multimedia media translations start',
+                [
+                    'multimediaId' => $multimedia->getId()->getValue(),
+                    'multimediaName' => $multimedia->getName(),
+                    'mediaId' => $shopware6Media->getId()
+                ]
+            );
+            $this->updateTranslations($channel, $shopware6Media);
+            $this->logger->info(
+                'Updating multimedia media translations end',
+                [
+                    'multimediaId' => $multimedia->getId()->getValue(),
+                    'multimediaName' => $multimedia->getName(),
+                    'mediaId' => $shopware6Media->getId()
+                ]
+            );
+        }
+    }
+
+    /**
+     * @throws GuzzleException
+     */
+    private function updateTranslations(Shopware6Channel $channel, Shopware6Media $shopware6Media): void
+    {
+        if (!empty($shopware6Media->getTranslations())) {
+            $action = new PatchMediaAction($shopware6Media);
+            $this->connector->execute($channel, $action);
+        }
+    }
+
+    /**
+     * @return Shopware6MediaTranslation[]
+     * @throws GuzzleException
+     */
+    private function getMediaTranslations(Shopware6Channel $channel, string $shopwareMediaId): array
+    {
+        $query = new Shopware6QueryBuilder();
+        $query->limit(1000);
+
+        $action = new GetMediaTranslations($query, $shopwareMediaId);
+
+        try {
+            return $this->connector->execute($channel, $action);
+        } catch (ClientException $exception) {
```

why we ignore this exception?

---

thank, you - fixed - passed to export errors